guide/14-deep-learning/dl-on-time-series-data.ipynb

```diff
@@ -0,0 +1,296 @@
+{
```

Comment:
<p>Replace "<strong>the </strong>single variable will be ..." with "<strong>a</strong> single variable will be ..."</p><p>Replace 'TabularDataObject'  with 'tabular data object'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/953/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>